pom.xml

```diff
@@ -5,21 +5,21 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.71</version>
+    <version>5.0</version>
+    <relativePath />
   </parent>
 
   <properties>
-    <jenkins.version>2.450</jenkins.version>
-    <!-- java11 required since 2.357:
-          https://www.jenkins.io/blog/2022/06/28/require-java-11/ -->
-    <java.version>11</java.version>
-    <configuration-as-code.version>1.53.1</configuration-as-code.version>
+    <java.version>17</java.version>
+    <jenkins.version>2.479.1</jenkins.version>
+    <skipAnimalSniffer>false</skipAnimalSniffer>
     <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
+    <configuration-as-code.version>1887.v9e47623cb_043</configuration-as-code.version>
```

Comment:
Jenkins switched to new version formatting: https://github.com/jenkinsci/configuration-as-code-plugin/releases/tag/1346.ve8cfa_3473c94